index.html

```diff
@@ -5,11 +5,42 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-    <link rel="stylesheet" href="/teras-mavr-cv/assets/css/reset.css">
-
     <title>Document</title>
+
+    <link rel="stylesheet" href="../teras-mavr-cv/assets/css/reset.css">
+    <link rel="stylesheet" href="../teras-mavr-cv/assets/css/style.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="assets/css/reset.css">
    <link rel="stylesheet" href="assets/css/style.css">
```
this is how your code structured